support/mesh/src/lib.rs

```diff
@@ -29,8 +29,6 @@ pub use mesh_channel::pipe;
 pub use mesh_channel::rpc;
 pub use mesh_channel::ChannelError;
 pub use mesh_channel::ChannelErrorKind;
-pub use mesh_channel::MpscReceiver;
```

Comment:
are the implementations of `Mpsc{Snder,Receiver}` still being used under-the-hood somewhere, or are they redundant after the changes to `Sender/Receiver`?

Reply:
They're redundant. I'll remove them soon.